src/integrations/prefect-dbt/prefect_dbt/cli/commands.py

```diff
@@ -156,8 +162,9 @@ def trigger_dbt_cli_command_flow():
         cli_args.append("--project-dir")
         cli_args.append(project_dir)
 
-    if command_kwargs:
-        cli_args.append(command_kwargs)
+    if extra_command_args:
+        for value in extra_command_args:
+            cli_args.append(value)
```

Comment:
Instead of changing the interface, could we update this logic to transform the extra kwargs into a list?
```suggestion
    if command_kwargs:
        for k, v in command_kwargs:
            cli_args.extend([k, v])
```

Reply:
Would this work for a command that looks like this? 
` dbt run --model foo_model `

Want to make sure that the hyphens are supported as users will most likely want to specify models/nodes by name/tag.

Reply:
Ah you're right. Passing them in as a list does seem like the best available option.